### DISCUSSION TITLE
Improve nested ordered list numbering, please

### DISCUSSION THREAD

**Discussion by DebbieAtSeam** [GitbookIO/community#653] [Feature Requests]
It would be a big improvement if nested ordered lists followed the standard conventions of using different numbering types at different levels, such as...

1. First level
  a.  Second level
    i. Third level
2. First level

Currently, nested ordered lists use Arabic numerals at all levels, which looks messy and confusing.

1. First level
  1.  Second level
    1. Third level
2. First level

**Comment by JaniceManwiller** [comment]
I would also love to see this - I had asked about it quite a while ago.

**Comment by getsnoopy** [comment]
This needs to be taken up seriously. It's such a small fix, but a huge improvement. Having the same numbering for nested lists looks almost unprofessional, as it seems like it's a mistake.

**Comment by denisemcg** [comment]
Agree! This is a big problem for me when many of my pages contain instructions.
